### PR TITLE
Update guide to 0.33

### PR DESCRIPTION
**Pull request by CodesOtakuYT**

### CODE REVIEW

File: content/guide/buffer_creation/buffer_creation.md [hunk 1]
```diff
@@ -41,15 +55,15 @@ which isn't expensive. You should get used to passing the device as a parameter,
...
+buffer](https://docs.rs/vulkano/0.32.0/vulkano/buffer/struct.BufferUsage.html) for, which can help the
 implementation perform some optimizations. Trying to use a buffer in a way that wasn't indicated in
 its constructor will result in an error. For the sake of the example, we just create a
 `BufferUsage` that supports being used as a uniform buffer.
 
 The third parameter indicates if the buffer should be CPU cached. This should rarely be true for most
 use cases, but in some cases where the application is writing data to the GPU through this buffer continuously,
 setting this parameter to true may yield some performance gain. This parameter should not be true if
-the user intends to read results from the GPU from this buffer as GPU changes may not reflect.
+the user intends to read results from the GPU from this buffer than GPU changes may not reflect.
```
Comment (marc0246) [inline content/guide/buffer_creation/buffer_creation.md:66]: This is probably a mistake? At least it doesn't seem to make grammatical sense now.

File: content/guide/buffer_creation/example_operation.md [hunk 2]
```diff
@@ -45,8 +47,21 @@ from 0 to 63. The `destination_content` iterator produces 64 values that are all
 In other words, once created the source buffer contains sixty-four values ranging from 0 to 63
 while the destination buffer contains sixty-four 0s.
 
-## Command buffers
+## Creating a command allocator
```
Comment (marc0246) [inline content/guide/buffer_creation/example_operation.md:50]:
```suggestion
## Creating a command buffer allocator
```

File: content/guide/buffer_creation/buffer_creation.md (hunk 1 above)
Comment (CodesOtakuYT) [reply content/guide/buffer_creation/buffer_creation.md:66]: Actually true

File: chapter_code/src/bin/buffer_creation.rs [hunk 3]
```diff
@@ -52,34 +56,40 @@ fn main() {
...
             ..Default::default()
         },
         false,
         destination_content,
     )
-    .expect("failed to create buffer");
+    .expect("failed to create destination buffer");
 
-    let mut builder = AutoCommandBufferBuilder::primary(
+    let command_allocator = StandardCommandBufferAllocator::new(
```
Comment (marc0246) [inline chapter_code/src/bin/buffer_creation.rs:86]:
```suggestion
    let command_buffer_allocator = StandardCommandBufferAllocator::new(
```

File: chapter_code/src/bin/buffer_creation.rs [hunk 4]
```diff
@@ -52,34 +56,40 @@ fn main() {
...
         device.clone(),
+        StandardCommandBufferAllocatorCreateInfo::default(),
+    );
+
+    let mut builder = AutoCommandBufferBuilder::primary(
+        &command_allocator,
```
Comment (marc0246) [inline chapter_code/src/bin/buffer_creation.rs:92]:
```suggestion
        &command_buffer_allocator,
```

**Review by AustinJ235 — APPROVED** [review]
Great work! :tada:

**Comment by hakolao** [comment]
Nice!

**Pull request merged**